qemu/tests/s390x_cpu_model_expansion.py

```diff
@@ -0,0 +1,36 @@
+from virttest import env_process
+
+
+def run(test, params, env):
+    """
+    Boot guest and query-cpu-model-expansion
+
+    :param test: QEMU test object.
+    :param params: Dictionary with test parameters.
+    :param env: Dictionary with the test environment.
+    """
+    params['start_vm'] = 'yes'
+    vm_name = params['main_vm']
+    env_process.preprocess_vm(test, params, env, vm_name)
```

Comment:
If nothing needs to be changed, you can set `start_vm = yes` in the cfg directly.

Reply:
solved